jit-application/src/main/resources/application.yml

```diff
@@ -139,3 +139,12 @@ logging:
     org:
       hibernate:
         SQL: DEBUG
+
+camel:
```

Comment:
this part should be merged with the main camel part, it should not be in the logsql profile

Reply:
Yup, my bad, I just let git do its thing, didn't notice that it separated it out. Fixed it.